0003-interledger-protocol/0003-interledger-protocol.md

```diff
@@ -147,61 +145,62 @@ Connectors implement the interledger protocol to forward payments between ledger
 
 Connectors also implement the [Connector to Connector Protocol (CCP)](../0010-connector-to-connector-protocol/) to coordinate routing and other interledger control information.
 
-### Errors
+### Packet
```

Comment:
I think we should refer to it as the ILP Payment Packet, because there are other types of ILP (related) packets. That's what @justmoon did in the ASN.1 description